app/src/main/java/com/todoist_android/ui/home/BottomSheetFragment.kt

```diff
@@ -175,8 +178,8 @@ class BottomSheetFragment(var addNewTaskCallback : ()->Unit ) : BottomSheetDialo
             title = title,
             id = loggedInUserId,
             description = description,
-            status = status,
-            dueDate = "${dueDate ?: " "} ${selectedTime ?: " "}"
+            dueDate = "${dueDate ?: " "} ${selectedTime ?: DateTimeFormatter.ofPattern("HH:mm:ss").format(LocalDateTime.now())}",
```

Comment:
The due date and selected time is already formatted

Reply:
backend throws an error when we don't pass a combination of both... I noticed if I add a task and don't select date/time and leave the app to set the default, it only sends the date without the time. We need to make sure selected time is set to current if user does set those. Don't know if that makes sense @Njumbi

Reply:
Alright, good catch